### ISSUE TITLE
NTR: Increased fecal porphyrin

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#5627]
**Preferred term label:**
Increased fecal porphyrin

<br>

**Synonyms**
Increased fecal coproporphyrin 1; Increased fecal coproporphyrin 3; Total fecal porphyrins

<br>

**Definition (free text, please give PubMed ID)**
Abnormally high concentration of fecal porphyrins in feces.

[PMID:25372274](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4279155/)  

Other resources:  
- [Rare Disease Network](https://www.rarediseasesnetwork.org/cms/porphyrias/Healthcare_Professionals/diagnosis)  
- [porphyriafoundation.org](https://porphyriafoundation.org/for-patients/about-porphyria/testing-for-porphyria/tests-for-porphyria-diagnosis/)

<br>

**Parent term (use hpo.jax.org/app)**
HP_0032483 (Abnormal fecal test result) 

_Alternative_: consider adding a new term that would be the subclass of _Abnormal fecal test_ to enable specification of the type of porphyrin detected, for example:

```bash
Abnormal fecal test result  
                   |
                   |
          Abnormal Fecal porphyrin
                              |
                              |
                     Increased fecal coproporphyrin 1
                     Increased fecal coproporphyrin 3
```

If you liked this route, I could submit separate NTRs for each proporphyrin type listed above.

<br>

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
>Total fecal porphyrins are markedly increased especially in active Hereditary Coproporphyria (HCP) and Variegate Porphyria (VP) and to a lesser extent in Porphyria Cutanea Tarda (PCT) and Erythropoietic Protoporphyria (EPP).

Concept | OMIM | Orphanet
:--: | :--:| :--:  
Hereditary Coproporphyria | [OMIM:121300](https://omim.org/entry/121300) | [ORPHA:79273](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=11299&Disease_Disease_Search_diseaseGroup=Hereditary-Coproporphyria&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Hereditary-coproporphyria&title=Hereditary%20coproporphyria&search=Disease_Search_Simple)  
Variegate Porphyria | [OMIM:176200](https://omim.org/entry/176200?search=Variegate%20Porphyria&highlight=porphyria%20variegate) | [ORPHA:79473](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=11499&Disease_Disease_Search_diseaseGroup=Variegate-Porphyria&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Porphyria-variegata&title=Porphyria%20variegata&search=Disease_Search_Simple)  
Porphyria Cutanea Tarda | [OMIM:176100](https://omim.org/entry/176100?search=Porphyria%20Cutanea%20Tard&highlight=cutanea%20porphyria%20tard) | [ORPHA:101330](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=14841&Disease_Disease_Search_diseaseGroup=Porphyria-Cutanea-Tarda&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Porphyria-cutanea-tarda&title=Porphyria%20cutanea%20tarda&search=Disease_Search_Simple) 
Erythropoietic Protoporphyria | [OMIM:177000](https://omim.org/entry/177000?search=Erythropoietic%20Protoporphyria&highlight=%28erythropoiesi%7Cerythropoietic%29%20protoporphyria) | [ORPHA:79278](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=11304&Disease_Disease_Search_diseaseGroup=Erythropoietic-Protoporphyria&Disease_Disease_Search_diseaseType=Pat&Disease(s)/group%20of%20diseases=Autosomal-erythropoietic-protoporphyria&title=Autosomal%20erythropoietic%20protoporphyria&search=Disease_Search_Simple)

<br>

**Your nano-attribution (ORCID)**  
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
Thanks, adding this term. Could you add the others as well? There seems to be some specificity.

**Comment by callahantiff** [comment]
> Thanks, adding this term. Could you add the others as well? There seems to be some specificity.

No problem. I will add them as new issues and link them to this.

**Comment by pnrobinson** [comment]
Closing, we will add the annotations after the next HPO release! thanks!
